### PR TITLE
Add a tool to analyze and compare test output files

### PR DESCRIPTION
**Pull request by gmarek**
cc @wojtek-t @fgrzadkowski

### CODE REVIEW

File: test/resultcompare/resultcompare.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+type percentileUsageData struct {
+	perc    int
+	cpuData []float64
+	memData []int64
+}
+
+type ViolatingDataPair struct {
+	perc       int
+	lhsCpuData []float64
```
Comment (mwielgus) [inline test/resultcompare/resultcompare.go]: left/right

**Comment by mwielgus** [comment]
LGTM

**Comment by gmarek** [comment]
#19230 
@k8s-bot unit test this

**Comment by gmarek** [comment]
Moved to https://github.com/kubernetes/contrib/pull/372

**Pull request closed without merging**